src/components/dashboards/EconomieEnCultuur.vue

```diff
@@ -156,6 +149,8 @@ export default {
 }
 </script>
 
-<!-- Add "scoped" attribute to limit CSS to this component only -->
 <style lang="scss" scoped>
+  .grid_4 {
```

Comment:
Not very descriptive, the classname does not tell what it does

Reply:
added a comment